src/MessagingBubble/MessagingBubble.tsx

```diff
@@ -26,29 +30,35 @@ export const MessagingBubble: React.FC<Props> = ({
   children,
   theme,
   replyTo,
+  attachments,
 }: Props) => {
+  const bubbleClassNames = cx(
+    cssClasses.MESSAGE_BASE,
+    theme === "ownMessage" ? cssClasses.MESSAGE_OWN : cssClasses.MESSAGE_OTHER,
+    replyTo && cssClasses.MESSAGE_REPLY_PARENT,
+  );
+
+  const replyClassNames = cx(
+    cssClasses.MESSAGE_BASE,
+    theme === "ownMessage" ? cssClasses.MESSAGE_REPLY_OWN : cssClasses.MESSAGE_REPLY_OTHER,
+  );
+
+  const attachmentClassNames = cx(
+    cssClasses.MESSAGE_BASE,
+    theme === "ownMessage"
+      ? cssClasses.MESSAGE_ATTACHMENT_OWN
+      : cssClasses.MESSAGE_ATTACHMENT_OTHER,
+  );
```

Comment:
I relocated some of the existing logic to simplify the markup now that the markup is getting a little more complex.